std/meta.zig

```diff
@@ -115,6 +115,66 @@ test "std.meta.Child" {
     testing.expect(Child(?u8) == u8);
 }
 
+/// Given an array/pointer type, return the slice type `[]Child`.
+/// Preserves all pointer attributes such as `const`/`volatile` etc.
+pub fn SliceType(comptime T: type) type {
```

Comment:
Is `Type` redundant?  Should it just be `Slice` and `ArrayPointer` rather than `SliceType` and `ArrayPointerType`?

Reply:
I prefer without the `Type` suffix since we have a pretty consistent formatting style that types are capitalized.